rust/yew/todomvc/static/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Yew • TodoMVC</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/todomvc-common@1.0.5/base.css"/ >
```

Comment:
Is it worth pulling in the css locally as well?

Reply:
I don't think it is worth. But can be done next if you feel it is best